lucene/core/src/java/org/apache/lucene/search/SortField.java

```diff
@@ -148,12 +149,12 @@ public Provider() {
 
     @Override
     public SortField readSortField(DataInput in) throws IOException {
```

Comment:
this one is tricky becase it's not versioned. Do you think we could remove all calls to `EndiannesReverserUtil` from it and instead fix callers to wrap the input with the EndiannessReverser wrapper?

Reply:
I introduce a wrapper to the DataOutput as well and now this classes remain unchanged.